reporting/src/request/index.js

```diff
@@ -55,7 +55,7 @@ export default class RequestMonitor {
       communication,
       getBrowserInfo,
       onTrackerInteraction = (event, state) => {
-        logger.log(`Tracker '${event}' with url: ${state.url}`);
+        logger.info(`Tracker '${event}' with url: ${state.url}`);
```

Comment:
```suggestion
        logger.info('Tracker', event, 'with url:', state.url');
```

Detail, but since the log will be normally be disabled in production, we can avoid constructing the string in patterns like that. It will be passed into "noop".